assets/stylesheets/layout/_main.scss

```diff
@@ -280,6 +280,13 @@ html.iframed {
 	line-height: 1;
 }
 
+/* Emoji and smileys */
+.emoji,
```

Comment:
If you all agree, I'll update this PR to remove those other styles, and have only this one.